packages/portfolio-deploy/src/access-token-setup.core.js

```diff
@@ -15,9 +15,9 @@ const trace = makeTracer('PoC-CE', true);
 /** @import {Board} from '@agoric/vats'; */
 
 const PoCInfo = /** @type {const} */ ({
-  issuerName: 'PoC25',
-  denom: 'upoc25',
-  decimalPlaces: 1,
+  issuerName: 'PoC26',
+  denom: 'upoc26',
+  decimalPlaces: 0,
```

Comment:
Updated decimalPlaces from 1 to 0, I forgot to separate this change into its own commit so thought I should point it out via comment